sphinx/ext/coverage.py

```diff
@@ -121,6 +124,14 @@ def write_c_coverage(self) -> None:
                 write_header(op, filename)
                 for typ, name in sorted(undoc):
                     op.write(' * %-50s [%9s]\n' % (name, typ))
+                    if self.config.coverage_print_missing_c_items:
+                        if self.app.quiet or self.app.warningiserror:
+                            logger.warning(__('undocumented c api: %s [%s] in file %s'),
+                                           name, typ, filename)
+                        else:
+                            logger.info(red('undocumented  ') + darkgreen('c   ') +
+                                        darkblue('api       ') + '%-30s' % (name + " [%9s]" % typ) +
+                                        red(' - in file ') + filename)
```

Comment:
I feel this message is too much colorized... Could you make this simple? We have to support who have poor eyesight (see #5989) in nearly future.

Reply:
Sure will try.

Reply:
![info log](https://user-images.githubusercontent.com/43916661/83355232-6da4ed80-a35e-11ea-970c-3f5737201eb8.PNG)

I remove the Colors on 2nd and 3rd columns. I think this is fine?

Reply:
LGTM!